### PR TITLE
[stable/atlantis] Atlantis v0.8.2

### PR DESCRIPTION
**Pull request by andyyaldoo**
#### What this PR does / why we need it:

Allow to pass `--disable-apply-all` server flag via `ATLANTIS_DISABLE_APPLY_ALL` env var

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/atlantis/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "v0.7.1"
+appVersion: "v0.8.0"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.5.3
+version: 3.6.1
```
Comment (andyyaldoo) [inline stable/atlantis/Chart.yaml]:
```suggestion
version: 3.6.0
```

**Comment by adharmawan** [comment]
@anubhavmishra @callmeradical Please kindly review :)

**Comment by adharmawan** [comment]
Hello @callmeradical and @anubhavmishra, could you look at this PR? 😄

**Comment by anubhavmishra** [comment]
@lkysow can you take a look at this?

File: stable/atlantis/README.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
...
 | `defaultTFVersion`                          | Default Terraform version to be used by atlantis server                                                                                                                                                                                                                                                   | n/a     |
 | `allowForkPRs`                              | Allow atlantis to run on fork Pull Requests                                                                                                                                                                                                                                                               | `false` |
+| `disableApplyAll`                           | disables running `atlantis apply` without any flags                                                                                                                                                                                                                                                       | `false` |
```
Comment (lkysow) [inline stable/atlantis/README.md]:
```suggestion
| `disableApplyAll`                           | Disables running `atlantis apply` without any flags                                                                                                                                                                                                                                                       | `false` |
```

**Review by lkysow — CHANGES_REQUESTED** [review]

**Comment by lkysow** [comment]
Thanks Andy, can you squash your commits so there's only 1? Then we're good to go.

**Comment by adharmawan** [comment]
Hi @lkysow, so I followed [this guide](https://github.com/wprig/wprig/wiki/How-to-squash-commits) on how to squash all the commits. Managed to do that (I only have one commit now)

```sh
❯ gst
On branch atlantisv0.8.0
Your branch and 'origin/atlantisv0.8.0' have diverged,
and have 1 and 479 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
```

but when I do a git log, it includes the commit messages from other charts.

```sh
commit eca4608f66c719b7b337b4ba412397275faa3d37 (HEAD -> atlantisv0.8.0)
Author: Andy Aldo Dharmawan <andyaldodharmawan@gmail.com>
Date:   Wed May 29 18:23:51 2019 +0800

    [stable/atlantis] Add options to pass --disable-apply-all

    Signed-off-by: Andy Aldo Dharmawan <andyaldodharmawan@gmail.com>

    [stable/atlantis] Update docker image tag

    Signed-off-by: Andy Aldo Dharmawan <andyaldodharmawan@gmail.com>

    [stable/atlantis] Bump version

    Signed-off-by: Andy Aldo Dharmawan <andyaldodharmawan@gmail.com>

    [stable/velero] Render `extraEnvVars` even when not using credentials (#14184)

    * [stable/velero] Render `extraEnvVars` even when not using credentials

    Fixes #14130

    Signed-off-by: Elisiano Petrini <elisiano@gmail.com>

    * [stable/velero] Chart version bump
...
```

I am pretty sure I'm doing it wrongly, would appreciate if you can advise on this

**Comment by lkysow** [comment]
The easiest thing for you to do is probably:
1. rename your branch locally to a backup: `git branch -m atlantisv0.8.0-backup`
1. get the latest helm/charts `master` and check out a new branch from that called `atlantisv0.8.0`
1. manually apply the same changes you made in this pull request
1. commit the changes in a single commit
1. force push your branch up

**Comment by adharmawan** [comment]
Done, thanks @lkysow

**Pull request merged**